### DISCUSSION TITLE
Multiple times classes in coloured segments

### DISCUSSION THREAD

**Discussion by ShiiRochi** [fomantic/Fomantic-UI#816] [General] [answered]
# Bug Report

I am trying to implement sided coloured segments and top border is something that breaks everything, so that I need to use !important, what I want to avoid.

## Steps to reproduce
1. create .ui.blue.segment
2. open Inspector
3. select it and see what classList is applied

## Expected result
.ui.blue.segment:not(.inverted) is applied

## Actual result
.ui.blue.segment.segment.segment.segment.segment:not(.inverted) is applied

## Testcase
https://jsfiddle.net/8hsq905t/5/

## Screenshot (when possible)
<img width="334" alt="Снимок экрана 2019-06-17 в 13 27 33" src="https://user-images.githubusercontent.com/10761254/59597953-be96e400-9103-11e9-8025-02434256a7a5.png">

## Version
2.7.6

## Theme
Default

**Comment by lubber-de** [comment, marked as answer]
The multiple assigned segment class is there to increase specificity without using `!important`, so also wanted behavior.
In your case you can do the same like
```css
.ui.ui.ui.sided.left.blue.segment:not(.inverted) {
 border-top: inherit;
}
```
See https://jsfiddle.net/6pxdewtL/

**Comment by ShiiRochi** [comment]
Ok, thanks, that’s acceptable solution
